docs/known_issues.md

```diff
@@ -92,3 +92,13 @@ Wicked configures the networking settings of the host based on the sysctl config
 net.ipv4.conf.all.forwarding=1
 net.ipv6.conf.all.forwarding=1
 ```
+
+## Canal and IP exhaustion
+
+By default Canal keeps track of pod IPs by creatig a lock file for each IP in `/var/lib/cni/networks/k8s-pod-network`. Each IP belongs to a single pod and will be deleted as soon as the pod is removed. However, in the unlikely event that containerd loses track of the running pods, lock files may be leaked and Canal will not be able to reuse those IPs anymore. If this occurs, you may experience IP exhaustion errors, for example:
+
+```bash
```

Comment:
nit: console or just no syntax for log messages; bash will highlight weird stuff if you use it for log output.
```suggestion
```console
```

Reply:
aaaaah ok, thanks!